network_info_modem_info.c

```diff
@@ -79,49 +79,113 @@ int network_info_add_to_map(QCBOREncodeContext *response_detail_map)
 	return 0;
 }
 
-int network_info_log(void)
+void modem_info_log_rsrp(void)
 {
-	LOG_DBG("====== Cell Network Info ======");
 	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_RSRP, sbuf, sizeof(sbuf));
 	LOG_DBG("Signal strength: %s", sbuf);
+}
 
+void modem_info_log_cur_band(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_CUR_BAND, sbuf, sizeof(sbuf));
 	LOG_DBG("Current LTE band: %s", sbuf);
+}
 
+void modem_info_log_sup_band(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_SUP_BAND, sbuf, sizeof(sbuf));
 	LOG_DBG("Supported LTE bands: %s", sbuf);
+}
 
+void modem_info_log_area_code(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_AREA_CODE, sbuf, sizeof(sbuf));
 	LOG_DBG("Tracking area code: %s", sbuf);
+}
 
+void modem_info_log_ue_mode(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_UE_MODE, sbuf, sizeof(sbuf));
 	LOG_DBG("Current mode: %s", sbuf);
+}
 
+void modem_info_log_operator(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_OPERATOR, sbuf, sizeof(sbuf));
 	LOG_DBG("Current operator name: %s", sbuf);
+}
 
+void modem_info_log_cell_id(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_CELLID, sbuf, sizeof(sbuf));
 	LOG_DBG("Cell ID of the device: %s", sbuf);
+}
 
+void modem_info_log_ip_address(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_IP_ADDRESS, sbuf, sizeof(sbuf));
 	LOG_DBG("IP address of the device: %s", sbuf);
+}
 
+void modem_info_log_fw_version(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_FW_VERSION, sbuf, sizeof(sbuf));
 	LOG_DBG("Modem firmware version: %s", sbuf);
+}
 
+void modem_info_log_lte_mode(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_LTE_MODE, sbuf, sizeof(sbuf));
 	LOG_DBG("LTE-M support mode: %s", sbuf);
+}
 
+void modem_info_log_nbiot_mode(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_NBIOT_MODE, sbuf, sizeof(sbuf));
 	LOG_DBG("NB-IoT support mode: %s", sbuf);
+}
 
+void modem_info_log_gps_mode(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_GPS_MODE, sbuf, sizeof(sbuf));
 	LOG_DBG("GPS support mode: %s", sbuf);
+}
 
+void modem_info_log_date_time(void)
+{
+	char sbuf[128];
 	modem_info_string_get(MODEM_INFO_DATE_TIME, sbuf, sizeof(sbuf));
 	LOG_DBG("Mobile network time and date: %s", sbuf);
+}
 
+int network_info_log(void)
```

Comment:
Currently `network_info_log` (and `network_info_add_to_map`) function signatures return `int`, but I don't think there is a case currently where they return anything besides `0`. Just wanted to double check that you're intentionally returning an `int` for these functions instead of `void`.

Reply:
It wasn't intentional, perhaps a future improvement can make some checks, perhaps return an error code if the modem info library wasn't initialized before calling this. I don't see this as a priority right now.